README.md

```diff
@@ -10,9 +10,9 @@ Steam for Linux requires the following:
 - 1 GHz Pentium 4 or AMD Opteron with 512 megabytes of RAM and 5 gigabytes of hard drive space, or better
 - Internet connection (Cable/DSL speeds recommended)
 - Ubuntu 12.04 LTS, fully updated
-- Latest graphics driver
+- the Latest graphics driver
```

Comment:
This might actually be worth changing, but it should be "The latest graphics drivers" then (plural in case multiple are required, sometimes that is the case.).